Description: #undef __GLIBC__ before defining it
Author: rmh
Upstream status: REJECTED.  DO NOT SUBMIT.  (send to debian instead)

---
 include/features.h |    1 +
 1 file changed, 1 insertion(+)

--- a/include/features.h
+++ b/include/features.h
@@ -352,6 +352,7 @@
 
 /* Major and minor version number of the GNU C library package.  Use
    these macros to test for features in specific releases.  */
+#undef __GLIBC__
 #define	__GLIBC__	2
 #define	__GLIBC_MINOR__	23
 
